timemachine/cpp/src/wrap_kernels.cpp

```diff
@@ -1202,23 +1202,30 @@ void declare_barostat(py::module &m) {
     std::string pyclass_name = std::string("MonteCarloBarostat");
     py::class_<Class, std::shared_ptr<Class>>(m, pyclass_name.c_str(), py::buffer_protocol(), py::dynamic_attr())
         .def(
-            py::init(
-                [](const int N,
-                   const double pressure,
-                   const double temperature,
-                   std::vector<std::vector<int>> group_idxs,
-                   const int frequency,
-                   std::vector<std::shared_ptr<timemachine::BoundPotential>> bps,
-                   const int seed) { return new Class(N, pressure, temperature, group_idxs, frequency, bps, seed); }),
+            py::init([](const int N,
+                        const double pressure,
+                        const double temperature,
+                        std::vector<std::vector<int>> group_idxs,
+                        const int frequency,
+                        std::vector<std::shared_ptr<timemachine::BoundPotential>> bps,
+                        const int seed,
+                        const bool adaptive) {
```

Comment:
nit[naming]: slight preference for more descriptive `adapt_proposal_scale` or similar https://github.com/proteneer/timemachine/blob/61e068297ce26083b1168535ba908f9987696605/timemachine/md/barostat/moves.py#L92

Reply:
Would `adapt_volume_scale_factor` be better here or should `volume_scale_factor -> proposal_scale_factor`?

Reply:
>Would adapt_volume_scale_factor be better here

No preference

>should volume_scale_factor -> proposal_scale_factor?

Current name `volume_scale_factor` is less ambiguous 👍

Reply:
Changed in [c08bd5c](https://github.com/proteneer/timemachine/pull/1152/commits/c08bd5c3e2f21532582f2ccdfdc7d97e86598edf)